data/templates/legacy-article.mst

```diff
@@ -1,6 +1,9 @@
 {{#css-files}}
 <link rel="stylesheet" type="text/css" href="resource:///com/endlessm/knowledge/data/templates/css/{{{.}}}"/>
 {{/css-files}}
+{{#custom-css-files}}
+<link rel="stylesheet" type="text/css" href="{{{.}}}"/>
+{{/custom-css-files}}
```

Comment:
Is this really necessary? it looks to me like adding to the wrapper is enough (?), and will simplify this patch.